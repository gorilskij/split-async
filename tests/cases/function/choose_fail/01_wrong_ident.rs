#![no_main]

use split_async::split;

#[split]
async fn foo() {}

#[split]
async fn bar() {
    choose!(baz)
}
