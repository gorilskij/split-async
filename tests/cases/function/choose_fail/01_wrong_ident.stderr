error[E0425]: cannot find value `sync_baz` in this scope
 --> tests/cases/function/choose_fail/01_wrong_ident.rs:8:1
  |
8 | #[split]
  | ^^^^^^^^ not found in this scope
  |
  = note: this error originates in the attribute macro `split` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0425]: cannot find value `async_baz` in this scope
 --> tests/cases/function/choose_fail/01_wrong_ident.rs:8:1
  |
8 | #[split]
  | ^^^^^^^^ not found in this scope
  |
  = note: this error originates in the attribute macro `split` (in Nightly builds, run with -Z macro-backtrace for more info)
